.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+Thank you for opening a Pull Request! 
+
+Please provide answers to the following questions, delete examples.
+
+--------------------------------------------------------------------------------
+
+## Does this PR concern a dataset or tutorial?
+
+(Example) Tutorial
+(Example) Dataset
+
+## What does this PR do?
+
+(Example) Adds a new tutorial
+(Example) Fixes an existing tutorial
+(Example) Adds a code sample for a dataset
+(Example) Adds code to process a dataset
+
+## Does this PR address an issue (what number)?
+
+(Example) Closes #123
+(Example) Addresses #123
+(Example) No
```

Comment:
In theory "No" shouldn't be an option, since contributors should always open a feature request or bug before submitting code. We include this in the tutorial process to ensure they don't waste time creating a PR that we aren't able to accept/review/merge.

Reply:
Done, removed the "no" example.

Agree, though don't necessarily want to decline a PR if not tied to an issue. E.g. a really minor editorial fix - faster for all if developer just submits the PR than to write issue, wait for okay and then submit PR.

We mention that a person needs to submit issue first in the Community tutorial guide, but not for fixes or datasets. I'd like to add a section to the README.md on contributions with these topics:

- Need to sign CLA
- If contributing new code they must have copyright and license
- Always submit an issue first
- If you'd like to submit a community tutorial, see these instructions
- Before editing or adding code to your folk, make sure that it is up to date here's how